cert/parameters/defaults.yaml

```
@@ -17,5 +17,6 @@ values:
   cert_mode: 644
   key_mode: 600
   remove: false
+  certs_update_cmd: ~
```

is it needed to insert this key with an empty value _(IIRC `~` is used to 'clean' a k/v in yaml_) to be able to use `{%- if mapdata.certs_update_cmd %}`? I would have thought that putting no value would be sufficient.

---

I'll test tomorrow but I think `yamllint` will not like having a key without value.

---

> I'll test tomorrow but I think `yamllint` will not like having a key without value.

@baby-gnu I'm a little wary of using `null`/`~` in YAML as well, since it doesn't always have the desired effect -- I would be more interested in it if it actually _unset_ the value altogether.  Anyway, wouldn't this work?

```suggestion
  certs_update_cmd: ""
```

---

I did that, thanks.